apps/condo/domains/meter/schema/RegisterMetersReadingsService.js

```diff
@@ -0,0 +1,257 @@
+        message: 'Invalid meter value',
+        messageForUser: 'api.meter.registerMetersReadings.INVALID_METER_VALUE',
+        messageInterpolation: { key, value },
+    }),
+}
+
+function toISO (str) {
+    return dayjs(str).toISOString()
+}
+
+/**
+ * @param value
+ * @return {undefined|string|null}
+ */
+function normalizeMeterValue (value) {
+    if (!(isString(value) || isNumber(value) || isUndefined(value))) return null
+    if (isUndefined(value)) return value
+
+    const transformedValue = String(value).trim().replaceAll(',', '.')
+    if (isEmpty(transformedValue)) return undefined
+    return String(Number(transformedValue))
+}
+
+/**
+ * @param {string | null | undefined} value
+ * @return {boolean}
+ */
+function validateMeterValue (value) {
+    if (!isString(value) && !isUndefined(value)) return false
+    if (isUndefined(value)) return true
+
+    return !isEmpty(value) && !isNaN(Number(value)) && isFinite(Number(value)) && Number(value) >= 0
+}
+
+const RegisterMetersReadingsService = new GQLCustomSchema('RegisterMetersReadingsService', {
+    types: [
+        {
+            access: true,
+            type: 'input RegisterMetersReadingsMeterMetaInput {' +
+                'numberOfTariffs: Int,' +
+                'place: String,' +
+                'verificationDate: String,' +
+                'nextVerificationDate: String,' +
+                'installationDate: String,' +
+                'commissioningDate: String,' +
+                'sealingDate: String,' +
+                'controlReadingsDate: String' +
+                '}',
+        },
+        {
+            access: true,
+            type: 'input RegisterMetersReadingsReadingAddressMetaInput { unitType: String!, unitName: String!, globalId: String }',
+        },
+        {
+            access: true,
+            type: 'input RegisterMetersReadingsReadingInput {' +
+                'address: String!,' +
+                'addressMeta: RegisterMetersReadingsReadingAddressMetaInput!,' +
+                'accountNumber: String!,' +
+                'meterNumber: String!,' +
+                'meterResource: MeterResourceWhereUniqueInput!,' +
+                'date: String!,' +
+                'value1: String!,' +
+                'value2: String,' +
+                'value3: String,' +
+                'value4: String,' +
+                'meterMeta: RegisterMetersReadingsMeterMetaInput' +
+                '}',
+        },
+        {
+            access: true,
+            type: 'input RegisterMetersReadingsInput { dv: Int!, sender: SenderFieldInput!, organization: OrganizationWhereUniqueInput!, readings: [RegisterMetersReadingsReadingInput!]! }',
+        },
+    ],
+
+    mutations: [
+        {
+            access: access.canRegisterMetersReadings,
+            schema: 'registerMetersReadings(data: RegisterMetersReadingsInput!): [MeterReading]',
+            resolver: async (parent, /**{ data: RegisterMetersReadingsInput }*/args, context) => {
```

Comment:
```suggestion
            resolver: async (parent, args, context) => {
```

Reply:
This is just for typehinting
<img width="809" alt="image" src="https://github.com/open-condo-software/condo/assets/25384290/d6b091af-ce63-4897-90db-5a09d4a4f22d">

Reply:
Yep, my bad)